docs/eventing/samples/cloudpubsubsource/README.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "CloudPubSubSource"
+linkTitle: "CloudPubSubSource"
+weight: 20
+type: "docs"
+---
+
+# CloudPubSubSource Example
```

Comment:
from the new install page we have a link to the old gcp-pub-sub-source example. Remember to point it to this one. Thanks!